frontend-react/vite.config.ts

```diff
@@ -8,6 +8,10 @@ import remarkGfm from "remark-gfm";
 import rehypeSlug from "rehype-slug";
 import remarkToc from "remark-toc";
 import { checker } from "vite-plugin-checker";
+import dotenv from "dotenv";
+
+// Force usage of custom environment files
+dotenv.config({ path: `.env.${process.env.NODE_ENV}` });
```

Comment:
Interesting 🤔 ... I thought this would be the default behavior

Reply:
I thought so too! It seems it's just prod/dev by default though, so you gotta force it.